### PR TITLE
fix(workbox-webpack-plugin): fix typo

### PR DESCRIPTION
**Pull request by aicest**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/GoogleChrome/workbox/blob/v5/packages/workbox-build/src/entry-points/options/common-generate-schema.js#L23  https://github.com/GoogleChrome/workbox/blob/v6/packages/workbox-webpack-plugin/src/generate-sw.js#L87
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/workbox-webpack-plugin/index.d.ts [hunk 1]
```diff
@@ -427,9 +427,9 @@ export interface GenerateSWOptions extends CommonOptions {
      * Those URL parameters would normally cause the cache lookup to fail, since the URL strings used as cache keys would not be expected to include them.
      *
      * @default [/^utm_/]
-     * @example ignoreUrlParametersMatching: [/./]
+     * @example ignoreURLParametersMatching: [/./]
      */
-    ignoreUrlParametersMatching?: RegExp[];
+    ignoreURLParametersMatching?: RegExp[];
```
Comment (aicest) [inline types/workbox-webpack-plugin/index.d.ts:432]: https://github.com/GoogleChrome/workbox/blob/v5/packages/workbox-build/src/entry-points/options/common-generate-schema.js#L23

Comment (aicest) [reply types/workbox-webpack-plugin/index.d.ts:432]: https://github.com/GoogleChrome/workbox/blob/v6/packages/workbox-webpack-plugin/src/generate-sw.js#L87

**Review by peterblazejewicz — APPROVED** [review]
@aicest thx for the fix!
LGTM!

**Comment by aicest** [comment]
Ready to merge

**Pull request merged**